.github/workflows/cli.yml

```diff
@@ -0,0 +1,42 @@
+name: CLI
+
+on:
+  schedule:
+    - cron: "0 18 * * 0"
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.11"]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - uses: azure/login@v2
```

Comment:
May need to add below content to make the login work (from https://github.com/marketplace/actions/azure-login#login-with-openid-connect-oidc-recommended)
```
permissions:
  id-token: write
  contents: read
```